esphome/components/modbus_controller/const.py

```diff
@@ -6,6 +6,7 @@
 CONF_FORCE_NEW_RANGE = "force_new_range"
 CONF_MODBUS_CONTROLLER_ID = "modbus_controller_id"
 CONF_MODBUS_FUNCTIONCODE = "modbus_functioncode"
+CONF_ON_COMMAND = "on_command"
```

Comment:
Probably better to be more specific here and say `on_command_sent`
I feel like `on_command` implies an incoming command

```suggestion
CONF_ON_COMMAND_SENT = "on_command_sent"
```

Reply:
I agree, I will make the changes